src/components/SEO.js

```diff
@@ -5,233 +5,239 @@ import Helmet from "react-helmet";
-      mainEntityOfPage: URL
-    };
-    // Push current blogpost into breadcrumb list
-    itemListElement.push({
-      "@type": "ListItem",
-      item: {
-        "@id": URL,
-        name: title
-      },
-      position: 3
-    });
-  }
-
-  const breadcrumb = {
-    "@context": "http://schema.org",
-    "@type": "BreadcrumbList",
-    description: "Breadcrumbs list",
-    name: "Breadcrumbs",
-    itemListElement
-  };
-
-  return (
-    <Helmet>
-      <html lang={config.siteLanguage} />
-      <title>{title}</title>
-      <meta name="keywords" content={keywords} />
-      <meta name="description" content={description} />
-      <meta name="image" content={image} />
-      <meta name="gatsby-starter" content="Gatsby Starter Minimal Blog" />
-      <meta property="og:locale" content={config.ogLanguage} />
-      <meta
-        property="og:site_name"
-        content={config.ogSiteName ? config.ogSiteName : ""}
-      />
-      <meta property="og:url" content={URL} />
-      <meta property="og:type" content={article ? "article" : "website"} />
-      <meta property="og:title" content={title} />
-      <meta property="og:description" content={description} />
-      <meta property="og:image" content={image} />
-      <meta property="og:image:height" content={"600"} />
-      <meta property="og:image:width" content={"600"} />
-      <meta property="og:image:alt" content={description} />
-      {config.siteFBAppID && (
-        <meta property="fb:app_id" content={config.siteFBAppID} />
-      )}
-      <meta name="twitter:card" content="summary_large_image" />
-      <meta
-        name="twitter:creator"
-        content={config.userTwitter ? config.userTwitter : ""}
-      />
-      <meta name="twitter:title" content={title} />
-      <meta name="twitter:url" content={config.siteUrl} />
-      <meta name="twitter:description" content={description} />
-      <meta name="twitter:image" content={image} />
-      <meta name="twitter:image:alt" content={description} />
-      {/* Insert schema.org data conditionally (webpage/article) + everytime (breadcrumbs) */}
-      {!article && (
-        <script type="application/ld+json">
-          {JSON.stringify(schemaOrgWebPage)}
-        </script>
-      )}
-      {article && (
-        <script type="application/ld+json">
-          {JSON.stringify(schemaArticle)}
-        </script>
-      )}
-      {/* <script type="application/ld+json">{JSON.stringify(breadcrumb)}</script> */}
-    </Helmet>
-  );
+	const { postNode, postPath, article, buildTime } = props;
+
+	let title = config.siteTitleAlt;
+	let description = config.siteDescription;
+	let keywords =
+		"Donate medical masks, covid19, homemade masks, homemade surgical mask, surgical mask, reuseable masks";
+
+	const realPrefix = config.pathPrefix === "/" ? "" : config.pathPrefix;
+	const homeURL = `${config.siteUrl}${realPrefix}`;
+	const URL = `${homeURL}${postPath || ""}`;
+	const image = `${homeURL}${config.siteBanner}`;
+
+	if (article) {
```

Comment:
@gvan12 What other meta fields should be supported?

Reply:
We could support additional images for pages by adding that to the frontmatter and using a fallback of the siteBanner. This will be more useful once we actually start using blog posts (currently named news I think). Blogs are _sort of_ supported in that way but beefing up the frontmatter/CMS integration is necessary beforehand.

Reply:
> We could support additional images for pages by adding that to the frontmatter and using a fallback of the siteBanner. 
Makes sense

What would the frontmatter look like:
```md
image2: /assets/whatever.jpg
image3: /assets/iamg3.jpg
```
?

Reply:
> > We could support additional images for pages by adding that to the frontmatter and using a fallback of the siteBanner.
> > Makes sense
> 
> What would the frontmatter look like:
> 
> ```lisp
> image2: /assets/whatever.jpg
> image3: /assets/iamg3.jpg
> ```
> 
> ?

I think we'd just have 1 image per page - so it'd pick that one first, then if none existed fall back to the `siteBanner`

Reply:
I added a featuredimage field as that's the name used in "News"